x-pack/plugins/security_solution/public/common/mock/mock_detail_item.ts

```diff
@@ -137,3 +137,5 @@ export const mockDetailItemData: TimelineEventsDetailsItem[] = [
     isObjectArray: false,
   },
 ];
+
+export const mockDetailItemData: TimelineEventsDetailsItem[] = generateMockDetailItemData();
```

Comment:
FYI: I wanted to refactor all usages of this static data to use the function created above, but opted instead to keep the scope small since I'm not that familiar with this area of the code. My goal in creating the above function is to avoid sharing state between test cases since the data can be mutated.